### ISSUE TITLE
Remove flatfield on zoom logic

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/dodal#203]
The logic for applying the flatfield to the OAV when it moves to zoom 1x has been moved into EPICS on i04 and soon to be on i03 (https://jira.diamond.ac.uk/browse/I03-912). Once this is merged we can remove the code from `dodal`

**Comment by stan-dot** [comment]
```python

    def set_flatfield_on_zoom_level_one(self, value):
        self.parent: OAV
        flat_applied = self.parent.proc.port_name.get()
        no_flat_applied = self.parent.cam.port_name.get()
        return self.parent.grid_snapshot.input_plugin.set(
            flat_applied if value == "1.0x" else no_flat_applied
        )
```

is it about this snippet from `dodal.devices.oav.oav_detector` ? 

the Jira ticket in question last activity is Nov'23 and marked as deployed but not yet accepted

**Comment by DominicOram** [comment]
Yes, it has actually now been deployed and so we do not need the code. This is actually a duplicate of https://github.com/DiamondLightSource/dodal/issues/801, which I have just done the work for so this can be closed.
